examples/topics/geo/tile_xyzservices.py

```diff
@@ -1,10 +1,28 @@
-import xyzservices.providers as xyz
+''' A geographical plot that demonstrates how to use XYZ tile services to create an interactive map.
+The xyzservices provides map tiles,which are typically used to display maps visualizations 
+and geographical location in web applications.The data used to create the plot is 
+sourced from public XYZ tile services.This example demonstrates how to create a plot in Bokeh
+with a map background using tile_providers.The chart shows a map of a specific geographical region, 
+which is defined by the range bounds supplied in web mercator coordinates.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.Figure, bokeh.tile_providers.get_provider, bokeh.tile_provider.Vendors
+    :refs: :ref:`ug_topics_geo_tile_provider_maps`
+    :keywords: Bokeh, tile services, XYZ, mapping, interactive, geography
+
 
+'''
+
+import xyzservices.providers as xyz
 from bokeh.plotting import figure, show
+from bokeh.tile_providers import get_provider, Vendors
 
 # range bounds supplied in web mercator coordinates
 p = figure(x_range=(-2000000, 6000000), y_range=(-1000000, 7000000),
            x_axis_type="mercator", y_axis_type="mercator")
-p.add_tile(xyz.OpenStreetMap.Mapnik)
+
+# Retrieve the OpenStreetMap.Mapnik tile source from the tile providers
+tile_provider = get_provider(Vendors.CARTODBPOSITRON)
+p.add_tile(tile_provider)
```

Comment:
All of the code changes should be reverted. This example is specifically for illustrating the use of the `xyxservices` package, so it needs to do that (as it did originally)